tests/app/celery/test_broadcast_message_tasks.py

```diff
@@ -59,3 +60,59 @@ def test_send_broadcast_message_errors(sample_service):
             send_broadcast_message(broadcast_message_id=str(bm.id))
 
     assert ex.value.response.status_code == 503
+
+
+@freeze_time('2020-08-01 12:00')
+def test_send_broadcast_event_sends_data_correctly(sample_service):
+    t = create_template(sample_service, BROADCAST_TYPE)
+    bm = create_broadcast_message(t, areas=['london'], status=BroadcastStatusType.BROADCASTING)
+    be = create_broadcast_event(bm)
+
+    with requests_mock.Mocker() as request_mock:
+        request_mock.post("http://test-cbc-proxy/broadcasts/events/stub-1", json={'valid': 'true'}, status_code=200)
+        send_broadcast_event(broadcast_event_id=str(be.id))
+
+    assert request_mock.call_count == 1
+    assert request_mock.request_history[0].method == 'POST'
+    assert request_mock.request_history[0].headers["Content-type"] == "application/json"
+
+    cbc_json = request_mock.request_history[0].json()
+    assert cbc_json['id'] == str(be.id)
+    assert cbc_json['broadcast_message_id'] == str(bm.id)
+    assert cbc_json['sent_at'] == '2020-08-01T12:00:00.000000Z'
+    assert cbc_json['transmitted_starts_at'] is None
+    assert cbc_json['transmitted_areas'] == ['london']
+
+
+def test_send_broadcast_event_sends_references(sample_service):
```

Comment:
we could also have an assertion that logging works correctly potentially ☺️

Reply:
we don't normally test logging messages, unless it a log message that we care about that changes (eg: a log message for an exception that we rely on to raise a pagerduty notification, or one that we specifically dont want to raise a pagerduty like in david's PR this morning) - is there something you specifically wanted to catch with these two log lines?

Reply:
no, makes sense, I try to test any log statements I put in, so there is less chance we silently stop getting those logs, but you ar probably right that it's not essential ☺️